src/components/confirmationComponents/OrderDetailsSection.tsx

```diff
@@ -0,0 +1,95 @@
+import {Image} from 'react-bootstrap'
+
+import {useAppSelector} from '../../redux/hooks'
+import {DishInCart} from '../../common/types/dishesType'
+
+import './orderDetails.scss'
+import DetailsData from './DetailsData'
+
+
+const OrderDetailsSection = () => {
+  const orderedDishes = useAppSelector<DishInCart[]>(
+    (state) => state.cart.dishes
+  )
+  const totalSum = orderedDishes.reduce(
+    (sum: number, current) =>
+      sum + Number(current.price) * current.numberOfDishes,
+    0
+  )
+
+  return (
+    <>
+      <div className='details-block'>
+        <div className='details-title'>
+         Заказ:
+        </div>
+        <div className='shadow details-border'>
+          <div className='details-item'>
+            <div className='
+              details-item-block
+              details-header
+              detail-header-name'>
+              <span>Название</span>
+            </div>
+            <div className='details-item-block details-header'>
+              <span>Количество</span>
+            </div>
+            <div className='details-item-block details-header'>
+              <span>Цена за 1 позицию</span>
+            </div>
+          </div>
+          <div className='details-devider'></div>
```

Comment:
Лучше написать самозакрывающийся тег: 
```suggestion
          <div className='details-devider' />
```

Reply:
что-то новенькое для меня. Раньше не встречала самозакрывающиеся дивы, думала это синтаксис только для кнопок и инпутов